src/typeahead/typeahead-container.component.ts

```diff
@@ -92,6 +102,7 @@ export class TypeaheadContainerComponent implements OnDestroy {
     public element: ElementRef,
     private changeDetectorRef: ChangeDetectorRef
   ) {
+    this.element.nativeElement.id = this.popupId;
```

Comment:
Please use **renderer** to add new attribute

Reply:
Ok, i make this